src/Session.hs

```diff
@@ -81,11 +81,15 @@ kill ghci = ignored $ do
     -- See: https://github.com/ndmitchell/ghcid/issues/254
     showCursor
 
-loadedModules :: [Load] -> [FilePath]
-loadedModules = nubOrd . map loadFile . filter (not . isLoadConfig)
```

Comment:
This was the only use of `isLoadConfig`, so you could remove it from `Language.Haskell.Ghcid.Types` ([link](https://github.com/ndmitchell/ghcid/blob/e2852979aa644c8fed92d46ab529d2c6c1c62b59/src/Language/Haskell/Ghcid/Types.hs#L74C1-L77)).

There is no `isEval` function to go with the `Eval` data constructor, so you wouldn't be disrupting a complete set of functions, if you will.

Reply:
The fact that `isLoadConfig` isn't used elsewhere isn't a big deal - it's still an exported function on a public type, so has general utility.